index.html

```diff
@@ -6,13 +6,85 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
     <!-- don't forget to link your styles -->
+    <link rel="stylesheet" type="text/css" href="styles/style.css" />
   </head>
   <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+
+
+<!-- Navigatie bar met plaatje  -->    
+<section>
+  <!-- Blauwe spotify bar  -->    
+<div>
+  <nav class="nav-bar">
+    <img class ="greenlogo" src=images/spotify-logo.png alt="Spotify Logo">
+    <ul class="nav-bar-ul">
+    <li><a href="#premium">Premium</a></li>
+    <li><a href="#discover">Discover</a></li>
+    <li><a href="#help">Help</a></li>
+    <li><a href="#download">Download</a></li>
+    </ul></nav>
+</div>
+
+<header class ="everyone">
+<h1>Music for everyone.</h1>
+<p>Spotify is now free on mobile, tablet and computer. Listen to the right music, wherever you are.</p>
+</header>
+</section>
+
+
+<section class="section2">
+<h2>What's on Spotify?</h2>
+
+<div>
+<article class="blok">
+<img src = "images/music-icon.png" alt="Music Icon"/> 
+<h3>Millions of Songs</h3>
+<p class="text">There are millions of songs on Spotify</p>
+</article>
+
+<article class="blok">
+  <img src = "images/high-quality-icon.png" alt="High Quality"/> 
+  <h3>HD Music</h3>
+  <p class="text">Listen to music as if you were listening live</p>
+
+</article>
+
+<article class="blok">
+  <img src = "images/devices-icon.png" alt="Devices"/>
+  <h3>Stream Everywhere</h3>
+  <p class="text">Stream music on your smartphone, tablet or computer</p>
+
+</article>
+</div>
+
+</section>
+
```

Comment:
It would be easier if you used `<img id="whitelogo" src="images/spotify-icon-white.png" alt="Icon white" />` as a background image for `<section id="green"`. Then you would use flexbox on `<div class= "drierij">` to get `<div class="textrij">` and `<img id="kanye" src="images/spotify-app.jpg" alt="Spotify app" />` to appear side by side better